src/components/OrgFile/components/AgendaModal/index.js

```diff
@@ -176,14 +176,13 @@ const mapStateToProps = (state) => {
   const file = state.org.present.getIn(['files', path]);
   const allFiles = state.org.present.get('files');
   const fileSettings = state.org.present.get('fileSettings');
-  const agendaStartOnWeekday = state.base.get('agendaStartOnWeekday');
   return {
     files: determineIncludedFiles(allFiles, fileSettings, path, 'includeInAgenda', false),
     todoKeywordSets: file.get('todoKeywordSets'),
     agendaTimeframe: state.base.get('agendaTimeframe'),
     agendaDefaultDeadlineDelayValue: state.base.get('agendaDefaultDeadlineDelayValue') || 5,
     agendaDefaultDeadlineDelayUnit: state.base.get('agendaDefaultDeadlineDelayUnit') || 'd',
-    agendaStartOnWeekday: agendaStartOnWeekday == null ? 1 : +agendaStartOnWeekday,
+    agendaStartOnWeekday: state.base.get('agendaStartOnWeekday') || 1,
```

Comment:
Since number 0 is falsy, I believe this will lead to a behavior where tapping Sunday will set it to Monday

Reply:
I don't have a better idea than checking for null. Maybe the store has a method for this? Otherwise, I would check the spec, if store.get() ever returns undefined. If not, I'd go with `=== null` and otherwise rely on the store to return a integer (i.e. no `+` hack).

Reply:
You are correct, of course!

Apologies for the extra work. I'm currently on day 7 of the monthly 7-day Zen retreat - I'll take that and blame the lack of sleep for missing the falsiness of 0. Thank you for the extra effort and the review :pray: